docs/api.md

```diff
@@ -3127,7 +3127,7 @@ const { chromium } = require('playwright');  // Or 'firefox' or 'webkit'.
     await dialog.dismiss();
     await browser.close();
   });
-  page.evaluate(() => alert('1'));
+  await page.evaluate(() => alert('1'));
```

Comment:
This will throw because browser is closed in the middle of it. No await was deliberate, but maybe we should add silent catch().